shared/src/main/scala/doodle/core/Image.scala

```diff
@@ -41,3 +41,4 @@ final case class Overlay(t: Image, b: Image) extends Image
 final case class At(at: Vec, i: Image) extends Image
 final case class ContextTransform(f: DrawingContext => DrawingContext, image: Image) extends Image
 trait Drawable extends Image { def draw: Image }
+final case class Path(start:MoveTo,elements:List[PathElement]) extends Image
```

Comment:
For this edit suggest putting a space after the `,` in the case class fields, to have the same style as the rest of the classes above it.